md/README.md

```diff
@@ -0,0 +1,17 @@
+
+# YARPC
+
+YARPC is a coherent set of transport and encoding agnostic RPC libraries.
+YARPC libraries in [Python][], [Node.js][], [Go][], and [Java][] ship with HTTP
+and [TChannel][] transports and JSON and [Thrift][] encodings, and provides a
+common core that can be used for these or alternate encodings and transports.
+Each implementation provides a test harness and test subject as a Docker image
+and uses [Crossdock][] to validate correctness and consistency.
```

Comment:
We need references to all example code in here, as well as the graduated tutorial to first working service (without config) as a foundation. We lost this in the old README.